RussianFilter/sections/specific.txt

```diff
@@ -46,6 +46,8 @@ overclockers.ru##a[href^="http://segodnia.club/"] ~ *
 shostka.info###news-inside > div[id="dont_copy"] + div[class][style]
 ||shostka.info/media/$image
 shostka.info##aside[id^="aside"] > div.sb-element.media
+gidonline.io##iframe[src*="acdnpro.com"]
+gidonline.io##.sbin > div[id][style*="cursor: pointer;"]
```

Comment:
on my end it is covered by `gidonline.io###s-side > div.sbin`

Reply:
I don't see `#s-side` element (same in Chrome)
![Screenshot 2020-05-19 at 21 34 12](https://user-images.githubusercontent.com/18536328/82365334-34dd4e00-9a19-11ea-91aa-9dd1a123a5e3.png)